src/tooltip/Tooltip.js

```diff
@@ -0,0 +1,215 @@
+import * as React from 'react';
+import { TouchableOpacity, Modal, View } from 'react-native';
+import NativeMethodsMixin from 'react-native/Libraries/Renderer/shims/NativeMethodsMixin';
+import PropTypes from 'prop-types';
+
+import Triangle from './Triangle';
+import { Colors, ScreenWidth, ScreenHeight, isIOS } from './helpers';
+import getTooltipCoordinate from './getTooltipCoordinate';
+
+class Tooltip extends React.PureComponent {
+  state = {
+    isVisible: false,
+    yOffset: 0,
+    xOffset: 0,
+    elementWidth: 0,
+    elementHeight: 0,
+  };
+
+  static defaultProps = {
```

Comment:
Move this to the bottom under the proptype definitions - `Tooltip.propTypes`. It's much easier for readability